src/index.html

```diff
@@ -11,12 +11,97 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of company MOYO"
+          class="moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item name-1">
```

Comment:
Redundant, so you need to fix it everywhere
```suggestion
          <li class="nav__item">
```